README.md

```diff
@@ -12,12 +12,12 @@ SkyPilot is a framework for easily running machine learning workloads[^1] on any
 
```

Comment:
Should we remove the word "easily" from L11? (since it's repeated again in L13)

Reply:
Good call, done.